pytest.ini

```diff
@@ -9,12 +9,3 @@ markers =
 # Ignore setup and teardown for the timeout
 timeout_func_only = True
```

Comment:
Let's delete this because it produces the following warning

```
========================================================= warnings summary ==========================================================
../.venv/lib/python3.9/site-packages/_pytest/config/__init__.py:1439
  /home/dl/source/elastic/beats/.venv/lib/python3.9/site-packages/_pytest/config/__init__.py:1439: PytestConfigWarning: Unknown config option: timeout_func_only
  
    self._warn_or_fail_if_strict(f"Unknown config option: {key}\n")

-- Docs: https://docs.pytest.org/en/stable/how-to/capture-warnings.html
```

Reply:
Also, let's move this file from `/pytest.ini` to `/.buildkite/pytest.ini` in order to avoid potential clashes with other Beats modules that use pytest.

Reply:
as discussed over zoom, let's see how much time we want to put into this. 
Once we see that we need to do this now, I can move and do the moving files around.